sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -381,7 +381,7 @@ public virtual Response JobStatus(Guid jobId, bool? showStats = null, RequestCon
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
```

Comment:
These are breaking. The LROs eventually return a result (except DELETEs if returning a 204 eventually) but use `Operation-Location` header to refer to a status monitor per guidelines: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations-with-status-monitor. Unfortunately in this case, they don't use the old method of returning both a 200 from the LRO to show the response model + the 202 to provide the `Operation-Location` header. Maybe we need to do this in the short term, but @mikekistler isn't crazy about that.

Reply:
resolved